### DISCUSSION TITLE
Edit Profile on mobile app

### DISCUSSION THREAD

**Discussion by KennyOliver** [community/community#3280] [Mobile]
A feature which allows a profile to be edited from the app, e.g. change profile picture

**Comment by iamcrisb** [comment]
why would you not have this feature on mobile? xD

**Reply by fniutta** [reply]
idk, but i'd like to have it

**Reply by realred78** [reply]
> idk, but i'd like to have it

**Reply by realred78** [reply]
> > idk, but i'd like to have it 
> 
>

**Reply by SebTheRed** [reply]
idk, but i'd like to have it

**Reply by dragonflies4utoo** [reply]
+1(270)493-9515
leeanncoleman72@gmail.com
Me too I want to know

**Reply by BXLPika1** [reply]
Yo, he leaked his phone number

**Reply by lucieweber838-cyber** [reply]
Dx

**Reply by rhymeswithmogul** [reply]
> why would you not have this feature on mobile? xD

I have a photo of me on my phone. It'd be nice to add it from the same device the photo is on. Not that getting to my computer is hard, mind you, but it is an extra step. 

PROTIP: if [your About Me page is a repository](https://docs.github.com/en/account-and-profile/how-tos/profile-customization/managing-your-profile-readme), then you can edit your profile _text_ from your phone.

**Comment by jfullstackdev** [comment]
the temporary solution for now is to have it on the browser of your mobile, log in there

**Reply by isaiahvita** [reply]
Has this been tested? I spent like 5 minutes trying to do this on ios web mobile (safari and google). No option appears obvious.

**Reply by jazzyfresh** [reply]
i am also unable to find the edit button in the mobile browser...

EDIT: ok, the mobile ui is not quite intuitive: from your profile page click Edit Profile > click on your profile picture > then you need to scroll way down without clicking any of the menu options (this is where i got lost) > scrolling past the very long menu will bring you to the Public Profile edit page, where you can edit your photo by clicking on the photo with a little pencil-shaped edit icon.

of course, it didn't matter because it doesn't accept photos larger than 1 MB and i don't know how to resize photos on the native photo app, so i had to go to the desktop anyways 🫠

**Comment by ahmedverex** [comment]
awesome

**Reply by Sir-K-Grind** [reply]
I love it

**Comment by h-exx** [comment]
4 years and still no way to edit my profile

**Reply by rnag** [reply]
It's hard guys, programming it hard....
